src/hooks/use-field-state.js

```diff
@@ -15,9 +16,9 @@ export default function useFieldState(field: string) {
   const { fields } = useFormState();
   const { errors, meta, values } = fields ?? {};
   const fieldState = {
-    error: errors?.[field] ?? null,
-    meta: meta?.[field] ?? {},
-    value: values?.[field]
+    error: errors ? get(errors, field) : null,
```

Comment:
With `get` there's no need to check if `errors` exists. You can also pass the fallback value as the third argument.

```suggestion
    error: get(errors, field, null),
```

Please do the same in the two lines below.